mdn-drafts/ScreenAdvanced.md

```diff
@@ -0,0 +1,104 @@
+---
+recipe: api-interface
+title: 'ScreenAdvanced'
+mdn_url: /en-US/docs/Web/API/ScreenAdvanced
+specifications: https://webscreens.github.io/window-placement/#screenadvanced
+browser_compatibility: api.ScreenAdvanced
+---
+
+## Description
+
+The `ScreenAdvanced` interface of the Window Placement API extends the
+`Screen` interface and provides additional per-screen information.
+
+The `ScreenAdvanced` interface is a child of `Screen` and inherits its members.
+
+## Constructor
+
+None.
+
+## Properties
+
+_Inherits properties from its parent, `Screen`._
+
+**`ScreenAdvanced.left`**
+
+Returns the distance in pixels to the left edge of the screen from the primary screen origin.
```

Comment:
Technically, all the places that reference "the primary screen origin" should be something like "a multi-screen origin (e.g. the primary screen's origin)"; the multi-screen origin may be different on some platforms in certain circumstances. Feel free to ignore if that's too in-depth for this audience.

Reply:
That sounds like something that needs to be covered, but not here. (There's an overview page for this API that needs to be written.) Can you please send me an email about this.

Reply:
Will do. It still might make sense to replace "primary screen origin" with "multi-screen origin", but it's not a blocker for landing this PR